src/api/services/auth/auth.py

```diff
@@ -45,6 +49,8 @@ def token_required(f):
     """Authenticate that a valid Token is present."""
     @wraps(f)
     def decorated(*args, **kwargs):
+        from manage import app
+        access_time = str(datetime.datetime.utcnow().time())
```

Comment:
Impressive work on the logger functionality @mutoromike however I was thinking if it is possible to make the `access_time` variable global since I've seen you have declared it twice in the same file